index.md

```diff
@@ -19,19 +19,24 @@ Specifically, our supporters have provided:
 
 Of note: Oftentimes the discussion of and efforts to address harassment and discrimination have focused narrowly on sexual harassment and gender discrimination. We strongly believe that we need comprehensively and intersectionally inclusive workplaces and policies that consider not just gender but also race, age, sexuality, disability status, family status, immigration status, and many other dimensions of identity and experience.
 
-**This directory is open-source.** Any firm can add and update their policies and point(s)-of-contact, and we hope that every major venture firm will participate.
+**This directory is open-source.** Any firm can add and update their policies and point(s)-of-contact, and we hope that every major venture firm will participate. We value progress over perfection.
 
 <p class="btn-wrapper">
   <a href="https://airtable.com/shrudsWooa1wa7sCm" class="btn btn-dark">Join #MovingForward</a>
 </p>
 
 ## [](#vcs-mf) VCs #MovingForward
-_This is a living document, and we value progress, not perfection._
+_We've taken action towards creating more diverse and inclusive workplaces by making our discrimination & harassment policies and reporting contacts available to the public._
```

Comment:
nitpick on wording, to be more consistent with elsewhere on the site: something like this? "We've taken action towards creating more diverse and inclusive workplaces by making our anti-harassment/discrimination policy and reporting contact(s) available to the public."